src/lib/expansion-panel/expansion-panel.ts

```diff
@@ -98,6 +107,9 @@ export class ExpansionPanelComponent extends BaseComponent implements IExpansion
   @coreProperty()
   public declare animationType: ExpansionPanelAnimationType;
 
+  @coreProperty()
+  public declare trigger: string;
```

Comment:
See other comment about potentially adding a `triggerElement` property so that devs can manually set the trigger element instance to avoid race condition issues. We have run into this in Angular for other similar properties and this is usually preferred due to the async nature of incremental rendering (which is just the process of adding an element, waiting a frame, then adding another element which causes the component initialization to need to wait for discovery). Using direct element references fixes this problem and gives a nice alternative usage.

Once again the Invoker Commands API will eventually supersede this pattern completely by inverting the communication.